src/assets/scss/application.scss

```diff
@@ -61,23 +60,18 @@ $govuk-assets-path: "/public/";
   @include govuk-responsive-margin(3, "top");
 }
 
-#confirmDetailsContinue {
+#confirmDetailsContinue-btn {
   @include govuk-responsive-margin(6, "top");
```

Comment:
If it's just one override shall we use [spacing override classes](https://design-system.service.gov.uk/styles/spacing/#static-spacing-override-classes) instead of defining our own?

Reply:
I see it's also used in tests - we should really be using `data-test` attributes for that instead of classes whose responsibility is styling and can be easily changed